### ISSUE TITLE
turn off "X selection unavailable in this frame" (mis)feature?

### ISSUE THREAD

**Issue by jwaldmann** [syl20bnr/spacemacs#5758]
When I run spacemacs in a console window (emacs -nw foo.bar in a gnome-terminal)  , the usual X copy/paste is broken. This is a major annoyance. How can I turn this off? (Or why should I not?)
I need this when I'm remotely logged in (ssh running in the terminal, and spacemacs on the remote machine).

**Comment by syl20bnr** [comment]
I'm not sure to understand your issue, what do you want to turn off exactly?
I worked remotly via SSH in an `emacs -nw` instance on XQuartz today and didn't notice the message you mentioned, I was able to copy/paste correctly provided I'm staying on the same machine.

**Comment by jwaldmann** [comment]
Not copy-paste inside emacs, but using the mouse (copy: move mouse over text while pressing left mouse button, paste: middle mouse) - between different X windows.

**Comment by TheBB** [comment]
Turn off `xterm-mouse-mode`?

**Comment by jwaldmann** [comment]
Ah! That does it. (and my question is a duplicate of https://github.com/syl20bnr/spacemacs/issues/4591)

**Comment by novadeviator** [comment]
sorry to comment on the closed issue, but #4591 is about scrolling and this one is about copy/pasting. the solutions there are about enabling the scroll-wheel.

i have solved this issue permanently by editing my ~/.spacemacs inside the  
(defun dotspacemacs/user-config () section by adding  xterm-mouse-mode -1:
```
(defun dotspacemacs/user-config ()
  "Configuration function for user code.                                                                                            
This function is called at the very end of Spacemacs initialization after                                                           
layers configuration.                                                                                                               
This is the place where most of your configurations should be done. Unless it is                                                    
explicitly specified that a variable should be set before a package is loaded,                                                      
you should place your code here."
  (unless window-system
    (xterm-mouse-mode -1))
  )
```
This really should be there by default?
